documentation/User-Documentation/Variables-List.md

```diff
@@ -59,10 +59,22 @@ the variable will resolve.
     
 ## Calculate mathematical expression: `math.calc`
 
-This variable allows you to perform a calculation based on other variables (for example point or objective variables) and resolves to the result of the specified calculation. The variable always starts with `math.calc:`, followed by the calculation which should be calculated. Supported operations are `+`, `-`, `*`, `/` and  `^`. You can use `( )` and `[ ]` braces and also calculate absolute values with `| |` (but don't use this in the command event as it splits the commands at every `|`). If you want to use variables in the calculation, don't put `%` around them. Additionally, you can use the round operator `~` to round everything left of it to the number of decimal digits given on the right. So `4+0.35~1` will produce `4.4` and `4.2~0` will produce `4`.
+This variable allows you to perform a calculation based on other variables (for example point or objective variables)
+and resolves to the result of the specified calculation. The variable always starts with `math.calc:`, followed by the
+calculation which should be calculated. Supported operations are `+`, `-`, `*`, `/` and  `^`. You can use `( )` and
+`[ ]` braces and also calculate absolute values with `| |` (but don't use this in the command event as it splits the
+commands at every `|`). Additionally, you can use the round operator `~` to round everything left of it to the number of
```

Comment:
I'dd suggest adding a comment to the wiki that nesting of absolute values is not supported
```suggestion
`[ ]` braces and also calculate absolute values with `| |`. But be careful, don't use absolute values in the command
event as it splits the commands at every `|` and don't nest them (`|4*|3-5|` wont work).
Additionally, you can use the round operator `~` to round everything left of it to the number of
```

Reply:
I improved it and also included how to properly nest absolute values.